Dockerfile

```diff
@@ -23,7 +25,8 @@ RUN poetry install --no-dev --no-interaction --no-ansi
 
 COPY . /src
 
-RUN useradd --no-create-home --system -s /bin/false --uid 5000 uwsgi
+RUN groupadd --gid $GID uwsgi \
+    && useradd --no-create-home --system -s /bin/false --uid $UID --gid $GID uwsgi
```

Comment:
When you exec into the container, it's a little weird not having a name. Maybe we should give it one?

```
$ docker exec -it a536b7b91577 /bin/bash
I have no name!@a536b7b91577:/src$ whoami
whoami: cannot find name for user ID 5000
I have no name!@a536b7b91577:/src$
```

Reply:
I'm very consufed by the missing name because both the `uwsgi` is specified as group and user name and in the container I just built the container and this is the output:
```
platipo:~/resources_api $ sudo docker run -p80:5000 -it --rm resources_api/uwsgi /bin/bash
uwsgi@e26da29e625a:/src$ whoami
uwsgi
```

Reply:
I ran my container with `make run`, I wonder if that has something to do with it?

Reply:
It does `docker-compose run -p 5000:5000 resources-api flask run -h 0.0.0.0`. If you do that, do you also find that you're missing the user?

https://github.com/OperationCode/resources_api/blob/34e7f7033cc6dbf559f0c057c16c51765c2cdb32/Makefile#L58

Reply:
I tried some solutions, but I have no clue how to solve this :cry:

Reply:
Ok when I have some time I'll see if I can solve it. Maybe it's worth merging and opening that as a separate issue... I'll want to give it one more good review before I determine that. Thanks for your patience on this